### ISSUE TITLE
getCameraIds() fails in Vimba 1.3

### ISSUE THREAD

**Issue by thgis** [morefigs/pymba#7]
Hi

Downloaded and installed PvAPI, Vimba and pymba and ran into a problem while executing vimba.getCameraIds()
Apparently in Vimba 1.3.0 the vimba dll call camerasList return error code -9 (VmbErrorMoreData) even though a zero is passed for the length of the array.
Not sure if this is new actually, but for now I just changed line 93 in vimba.py to:
            if errorCode != 0 and errorCode!= -9: # The last check is for allowing incorrect array size error to be ignored
and I can sucessfully discover cameras.

**Comment by morefigs** [comment]
Do you mean it only happens in v1.3.0 and not previous versions? Does it happen if you only have one camera attached? Do you get the error on the second check as well (line 108), which should have the correct list length?

**Comment by thgis** [comment]
I dont have access to a camera until January the 5th so I cant do testing right now.
I only have one camera attached.
I believe I ran pymba with no problems before upgrading to vimba 1.3.0.
It seems that the problem is only the way that camerasList is called. The first call to this function with a 0 length array returned the -9 error BUT the actual number of cameras found is still written to numFound as expected. The second call to camerasList with a correctly sized array succeeds and camera ID is returned is the provided array.
So by ignoring error -9 in the first call to camerasList seems to be the only change needed.
And just one more note. I saw that they changed the manual entry for the function camerasList from vimba version 1.2.0 to 1.3.0. I take that as an indication that they might have changed the code behind the function. I cant remember or check the exact change right now.
